rhel/usr_lib_systemd_system_ovsdb-server.service

```diff
@@ -23,3 +23,4 @@ ExecReload=/usr/share/openvswitch/scripts/ovs-ctl --no-ovs-vswitchd \
            --no-monitor restart $OPTIONS
 RuntimeDirectory=openvswitch
 RuntimeDirectoryMode=0755
+RuntimeDirectoryPreserve=no
```

Comment:
We need to have this either as 'yes', or 'restart' - OVN daemons depend on this directory persisting even when the OVS daemons go away.

Reply:
Since the default is "RuntimeDirectoryPreserve=no", it's probably useless to specify it.
Moreover RuntimeDirectoryPreserve directive is not supported, and so ignored, on old systemd versions (< 235), including the RHEL7 one